openedx_events/content_authoring/data.py

```diff
@@ -0,0 +1,64 @@
+"""
+Data attributes for events within the architecture subdomain ``content_authoring``.
+
+These attributes follow the form of attr objects specified in OEP-49 data
+pattern.
+
+The attributes for the events come from the CourseDetailView in the LMS, with some unused fields removed
+(see deprecation proposal at https://github.com/openedx/public-engineering/issues/160)
+"""
+from datetime import datetime
+
+import attr
+from opaque_keys.edx.keys import CourseKey
+
+
+@attr.s(frozen=True)
+class CourseScheduleData:
+    """
+    Data describing course scheduling.
+
+    Arguments:
+        start (datetime): course start date
+        pacing (str): 'instructor' or 'self'
+        end (datetime): course end date (optional)
```

Comment:
I noticed a change to move `pacing` to be between `start` and `end`, which I think makes it a little harder to read. Is there something I'm missing?

Reply:
Note: I was going to write a similar comment, and then realized it was probably grouping required vs optional (which wasn't done on the other data class). I decided to not comment, but keeping start/end together, which share the same type would also make sense to me.

Reply:
non-optional arguments before optional ones